src/main/java/edu/pdx/cs/sftp/User.java

```diff
@@ -103,6 +78,31 @@ boolean verifyUsername(String usernameToVerify) {
     return matcher.matches();
   }
 
+  /**
+   * Prompts the user for a valid password. Password must not be empty or include spaces.
+   *
+   * @return validated password.
+   */
+  String getPassword() {
+    out.println("Enter your password:");
+    password = scanner.nextLine();
+    while (!verifyPassword(password)) {
+      err.println("You did not enter a password.\n" + "Please, enter your password:");
```

Comment:
```suggestion
      err.println("You did not enter a valid password.\n" + "Please, enter your password:");
```
They could have entered a pw with a space inside

Reply:
Hey! Originally, I thought about it but kept it because the password, unlike hostname and username, didn't have any requirements so if someone didn't enter anything and it would just say "you didn't entered a password" unlike like the others where it says "you didn't enter a valid password." Either way I think it's fine so I will commit your suggestion  so there is consistency :)